zerver/actions/user_settings.py

```diff
@@ -52,6 +52,42 @@ def send_user_email_update_event(user_profile: UserProfile) -> None:
     )
 
 
+def send_event_for_updating_delivery_email_fields(
+    user_profile: UserProfile, old_visibility_setting: int, visibility_setting: int
+) -> None:
+    active_users = user_profile.realm.get_active_users()
+    delivery_email_visible_user_ids = []
+    delivery_email_not_visible_user_ids = []
```

Comment:
`delivery_email_now_visible_user_ids` and `delivery_email_now_invisible_user_ids` might be better variable names, given that we're not doing anything special for ones that haven't changed?

Reply:
Updated.